aula-pratica-03/ex6.c

```diff
@@ -0,0 +1,22 @@
+#include<stdio.h>
+
+int main()
+{
+    int vetor[10], x;
+    for(int j = 0; j < 10; j++){
+        printf("Digite um numero: ");
+        scanf("%d", &vetor[j]);
+    }
+
+    printf("Digite um numero para buscar no vetor: ");
+    scanf("%d", &x);
+
+    for(int i = 0; i < 10; i++) {
+        if (x == vetor[i]){
+            printf("O numero foi encontrado na posicao [%d] do vetor", i);
+            return 1;
```

Comment:
I think it would be nice if you return 0 here and 1 if the value wasn't found :smile:

Reply:
bom, isso derrota o propósito 😂 , 
```suggestion
            return 0;
```